components/DetailPageHeader.js

```diff
@@ -152,6 +162,12 @@ class DetailPageHeader extends Component {
               {' '}
               <span className="bx--detail-page-header-status-text">
                 {statusText}{statusContent}
+                <span className="bx--detail-page-header-tag">
```

Comment:
I know it is not ideal to nest the `Tag` and `Link` children inside the `bx--detail-page-header-status-text`, but I did so to keep the elements inline without breaking backwards compatibility by renaming the span className.